python/cac_tripplanner/templates/admin/base.html

```diff
@@ -1,4 +1,19 @@
 {% extends "admin/base.html" %}
+{% load i18n %}
+{% block breadcrumbs %}
+  {% if title != 'Site administration' %}
+    <div class="breadcrumbs">
+      <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+      {% if title %}
+          {% if title|slice:'-14:' == 'administration' %}
+              &rsaquo; {{ title|slice:':-14' }}
+          {% else %}
+              &rsaquo; {{ title }}
+          {% endif %}
+      {% endif %}
+    </div>
+  {% endif %}
+{% endblock %}
 {% load cropping thumbnail %}
```

Comment:
There were differences in how the breadcrumbs were displayed between Django versions, and the `django-wpadmin` config doesn't allow any styling beyond a general display/hide boolean. This is a small fix that didn't feel worth applying to the forked package, cutting a release, and then updating the tag here (and we already had a template extending its `base.html`) so I applied the necessary display logic here.